abs.yaml

```
@@ -0,0 +1,6 @@
+# the conda build parameters to use
+build_parameters:
+  - "--variants \"{skip_cuda_prefect: True}\""
```

```suggestion
  - "--suppress-variables"
  - "--skip-existing"
  - "--error-overlinking"
  - "--variants \"{skip_cuda_prefect: True}\""
```
Because we still want to keep the default flags.